models/src/main/java/com/vimeo/networking2/LiveTime.kt

```diff
@@ -0,0 +1,23 @@
+package com.vimeo.networking2
+
+/**
+ * Live time data.
+ */
+data class LiveTime(
+
+    /**
+     * The amount of time per event that the user is allowed to live stream.
+     */
+    val eventMaximum: Int? = null,
```

Comment:
as this represents time, should this be a `Long` like the `monthlyMaximum` and `monthlyRemaining`?